config/coreboot-parrot.config

```diff
@@ -0,0 +1,20 @@
+CONFIG_TPM_MEASURED_BOOT=y
+CONFIG_VENDOR_GOOGLE=y
+CONFIG_BOARD_GOOGLE_PARROT=y
+CONFIG_CBFS_SIZE=0x780000
+CONFIG_BOARD_GOOGLE_PARROT=y
+CONFIG_IFD_BIN_PATH="../../blobs/parrot/ifd_shrinked.bin"
+CONFIG_ME_BIN_PATH="../../blobs/parrot/me_shrinked.bin"
+CONFIG_NATIVE_RAMINIT_IGNORE_MAX_MEM_FUSES=y
+CONFIG_NO_POST=y
+CONFIG_TPM_RDRESP_NEED_DELAY=y
+CONFIG_UART_PCI_ADDR=0
+CONFIG_NO_GFX_INIT=y
+CONFIG_CONSOLE_CBMEM_BUFFER_SIZE=0x80000
+CONFIG_DEFAULT_CONSOLE_LOGLEVEL_5=y
+CONFIG_PAYLOAD_LINUX=y
+CONFIG_PAYLOAD_FILE="../../build/Parrot/bzImage"
+CONFIG_LINUX_COMMAND_LINE="intel_iommu=igfx_off quiet"
+CONFIG_LINUX_INITRD="../../build/Parrot/initrd.cpio.xz"
+CONFIG_USE_OPTION_TABLE=y
+CONFIG_STATIC_OPTION_TABLE=y
```

Comment:
Wait.... That works?
That was a theoritical blocker for https://github.com/osresearch/heads/pull/944

So this boots? Not sure its required here unless the board requires users to modify  the defaults locally, as required per some users under t430

Reply:
So remove those please.

Reply:
done